ci/scripts/c_glib_build.sh

```diff
@@ -35,8 +36,16 @@ export CXXFLAGS="-DARROW_NO_DEPRECATED_API"
 
 mkdir -p ${build_dir}
 
+if [ "${BUILD_C_GLIB_WITH_VCPKG}" == "ON" ]; then
```

Comment:
Can we use existing `${VCPKG_ROOT}` instead of new variable?

```suggestion
if [ -n "${VCPKG_ROOT:-}" ]; then
```

Reply:
Yeah, I thought there might be cases where VCPKG_ROOT is defined but we don't necessarily want to use vcpkg, but I guess that's not very likely to happen.